automation/01-install-rucio.bash

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env bash
+set -e
+
+# Minikube
+# Possible values for memory: max | format: <number>[<unit>], where unit = b, k, m or g, e.g., 4000mb
+MINIKUBE_MEMORY_RAM=8000mb
```

Comment:
is this possible to override with env variables?

Reply:
Fixed in the commit fb245e3199a58021adda29142e66c639e8c64d06.

Furthermore, I added a help parameter for the script: `./01-install-rucio.bash -h` or `./01-install-rucio.bash --help`.

```text
▄▄▄▄▄▄▄▄▄▄▄▄
█   HELP   █
▀▀▀▀▀▀▀▀▀▀▀▀

Usage ▶ ./01-install-rucio.bash

Minikube
════════
Minikube accepts two parameters, the amount of memory and the number of CPUs.
  Memory:
    - max
    - format: <number>[<unit>], where unit = b, k, m or g, e.g., 4000mb
  CPUs:
    - max
    - 1, 2, 3, 4, 5, …, e.g., 8

  Usage
  ━━━━━
  To set up these parameters, need to use environment variables: MINIKUBE_MEMORY or MINIKUBE_CPU.
  Usage ▶ export MINIKUBE_MEMORY=5000mb MINIKUBE_CPU=3; ./01-install-rucio.bash
  Usage ▶ export MINIKUBE_CPU=max; ./01-install-rucio.bash
  Usage ▶ export MINIKUBE_MEMORY=max; ./01-install-rucio.bash

  Clean environment variables
  ━━━━━━━━━━━━━━━━━━━━━━━━━━━
  The unset command, removes the environment variable
  Usage ▶ unset MINIKUBE_MEMORY MINIKUBE_CPU

  Actual value for environment variables
  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
  Currently, these are the parameters which will set in the minikube. Blank means not arguments.
  ▶ minikube start --cpus=max
```